README.md

```diff
@@ -452,6 +453,14 @@ You can use the following environment variables to proxy HTTP and HTTPS requests
 
 When HTTP_PROXY / http_proxy are set, they will be used to proxy non-SSL requests that do not have an explicit proxy configuration option present. Similarly, HTTPS_PROXY / https_proxy will be respected for SSL requests that do not have an explicit proxy configuration option. It is valid to define a proxy in one of the environment variables, but then override it for a specific request, using the proxy configuration option.
 
+### Getting Supported APIs
+You can programatically obtain the list of supported APIs, and all available versions:
+
+```js
+const {google} = require('googleapis');
+const apiList = google.getSupportedAPIs();
```

Comment:
it's not a `[list]` actually, the name is misguiding here and everywhere.

Reply:
Fair enough 😜